### ISSUE TITLE
MemFC = 3 × MemW + 2 × Memlogits

### ISSUE THREAD

**Issue by eeric** [deepinsight/insightface#1989]
Partial FC: Training 10 Million Identities on a Single Machine
What does the formula 3 and 2 means in the formula?

**Comment by mnikitin** [comment]
@eeric during training classifier uses memory for weights (centroids), gradient and momentum (in case of sgd-like optimizer).
2 x Memlogts = logits + gradient

**Comment by eeric** [comment]
thank you
